src/components/organims/privacy/PrivacyLogin.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { StaticQuery, graphql } from "gatsby";
+import styled from 'react-emotion';
+import { css } from 'emotion';
+import { rem } from 'polished';
+
+import * as font from '../../../styles/fonts';
+import { mqMin } from '../../../styles/breackpoint';
+
+import SectionMinimal from '../../molecules/SectionMinimal';
+import Title from '../../atoms/Title';
+
+const boxWidth = 300;
+const boxHeight = 502;
+const sharedStyle = {
+  display: 'inline-block',
+  height: `${boxHeight}px`,
+  width: `${boxWidth}px`,
+  verticalAlign: 'middle',
+}
+
+const LoginBox = styled.div({
+  ...sharedStyle,
+  boxShadow: 'rgba(0, 0, 0, 0.20) 0px 4px 12px 0px',
+  borderRadius: rem('5px'),
+});
+const LoadingMessage = styled.div({
+  ...sharedStyle,
+  display: 'inline-block',
+  lineHeight: `${boxHeight}px`,
+  marginLeft: `-${boxWidth}px`,
+});
+
+class PrivacyLogin extends React.Component {
```

Comment:
For the future, using React function components could help writing leaner code, even more with the `useStaticQuery`.

Reply:
True, I realized after the fact that `useStaticQuery` was working only with function components, and that it's actually React's privileged way. Will do more of that in the future 👍